sparkbox/apprentices-page-next
# feat: Page descriptions support multiple apprenticships and application cycles

## Description

* Updated the copy of the main description to match our revision document.
* Changed Footer CTA to point to evergreen resources (careers page, newsletter sign up).
* Ran `npm audit fix` for security patches.

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. Review the paragraph next to the triangle illustration for typos.
5. Review the paragraph in the new footer for typos. 
6. Make sure the buttons work.
